tests/e2e_tests/test_data_compatibility.py

```diff
@@ -32,7 +43,15 @@ class TestStorageCompatibility:
 
     @staticmethod
     def _download_compatibility_data(version: str, storage_test_dir: Path):
-        """Download compatibility data for a specific version."""
+        """Download compatibility data for a specific version.
```

Comment:
@tellet-q this function seems to always download the archive without looking if it is already present on disk.

I thought the test grouping was enabling to reuse the downloaded archive.
How does it work exactly?